examples/ecs/ecs_guide.rs

```diff
@@ -224,8 +224,9 @@ fn exclusive_player_system(world: &mut World) {
 }
 
 // Sometimes systems need to be stateful. Bevy's ECS provides the `Local` system parameter
-// for this case. A `Local<T>` refers to a value owned by the system of type `T`, which is automatically
-// initialized using `T`'s `FromWorld`* implementation. In this system's `Local` (`counter`), `T` is `u32`.
+// for this case. A `Local<T>` refers to a value of type `T` that is owned by the system.
+// This value is automatically initialized using `T`'s `FromWorld`* implementation.
```

Comment:
```suggestion
// This value is automatically initialized using `T`'s `FromWorld`* implementation upon the system's initialization.
```

This might reformatting afterwards, but it's important to note *when* it's initialized.